biome.json

```
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.5.3/schema.json",
+	"organizeImports": {
+		"enabled": true
+	},
+	"formatter": {
+		"enabled": true
```

formatの設定を追加して、formatしなおしておいてほしいです
```suggestion
		"enabled": true,
                "indentWidth": 2,
                "indentStyle": "space"
```

---

@yuichimukai 
対応しました！

b16dff3bceb812f5cd014ff1374211e44a3b434f